redux-form/lib/actions.d.ts

```diff
@@ -138,7 +134,3 @@ export function unregisterField(form: string, name: string): Action;
  * Resets the 'touched' flag for all the fields passed in.
  */
 export function untouch(form: string, ...fields: string[]): Action;
-
-export function updateSyncErrors(from: string, syncErrors: FormErrors<FormData>, error: any): Action;
-
-export function updateSyncWarnings(form: string, syncWarnings: FormWarnings<FormData>, warning: any): Action;
```

Comment:
Why are `setSubmitFailed`, `setSubmitSucceeded`, `updateSyncErrors`, and `updateSyncWarnings` removed?